### PR TITLE
Added database file to admission_rules.d

### PR DESCRIPTION
**Pull request by stratos-karas**

### CODE REVIEW

File: oar-iccs/common_config.nix [hunk 1]
```diff
@@ -7,6 +7,7 @@ let
       propagatedBuildInputs = prev.propagatedBuildInputs ++ ([ pkgs.python3Packages.joblib pkgs.python3Packages.numpy pkgs.python3Packages.pandas pkgs.python3Packages.scikit-learn ]);
       postInstall = prev.postInstall + ''
       cp etc/oar/admission_rules.d/trainedGradientBoostingRegressor.model $out/admission_rules.d
```
Comment (nikosT) [inline oar-iccs/common_config.nix]: better add && at the end

File: oar-iccs/setup.toml [hunk 2]
```diff
@@ -7,7 +7,7 @@ kernel-params = "systemd.unified_cgroup_hierarchy=0"
```
Comment (nikosT) [inline oar-iccs/setup.toml]: remove file from pr

**Review by nikosT — CHANGES_REQUESTED** [review]

File: oar-iccs/common_config.nix (hunk 1 above)
Comment (stratos-karas) [reply oar-iccs/common_config.nix]: Added

File: oar-iccs/setup.toml (hunk 2 above)
Comment (stratos-karas) [reply oar-iccs/setup.toml]: File was removed

**Review by nikosT — APPROVED** [review]
good to go

**Pull request merged**